### DISCUSSION TITLE
Filequeue

### DISCUSSION THREAD

**Discussion by JFSolution-old** [OctoFarm/OctoFarm#212] [Feature Request]
A queue for files to print, where you can add new files which you want to print, but can‘t print because all printers are at work.
Change the order of the files by drag and drop, maybe set a priority flag (so other users don‘t change the order).

**Comment by NotExpectedYet** [comment]
This is already planned but it won't be part of the files view. It's going to be above that and compromised as Jobs/Queues. It's quite limiting to the other requests I've had to focus this on a list of files :).

**Reply by JFSolution-old** [reply]
Sounds good
